config/initializers/cors.rb

```diff
@@ -7,7 +7,7 @@
 
 Rails.application.config.middleware.insert_before 0, Rack::Cors do
   allow do
-    origins 'http://localhost:3000'
+    origins '*'
```

Comment:
This was not the issue before, it was mostly that I expected to see the JWT token within the response. From what I see, the token is within the headers, so this time I was able to locate it.